src/Resources/app/administration/src/module/blog-module/page/sas-blog-detail/index.js

```diff
@@ -211,5 +222,30 @@ Component.register('sas-blog-detail', {
         openMediaSidebar() {
             this.$parent.$parent.$parent.$parent.$refs.mediaSidebarItem.openContent();
         },
+
+        slugDetailsPage(result,value){
+            if (result.length > 0) {
+                if (result[0]['slug'] !== this.slugBlog) {
+                    this.blog.slug = value + "-" + "1"
+                }else {
+                    this.blog.slug = slugify(value, {
+                        lower: true
+                    });
+                }
+            }else {
+                this.blog.slug = slugify(value, {
+                    lower: true
+                });
+            }
+        },
+        slugCreatePage(result,value){
+            if (result.length > 0) {
+                this.blog.slug = value + "-" + "1"
+            }else {
+                this.blog.slug = slugify(value, {
+                    lower: true
+                });
+            }
+        }
```

Comment:
Fix it to the same the comments above

restrict the use of else, use return instead of else and make the code clear and shorter 

```suggestion
        slugDetailsPage(result, value) {
            if (result.length <= 0) {
                this.blog.slug = slugify(value, { lower: true });
                
                return;
            }
            
            if (result[0]['slug'] !== this.slugBlog) {
                    this.blog.slug = value + "-" + "1";
            } 
        },
        
        slugCreatePage(result, value) {
            ....
        }
```